src/cascadia/TerminalControl/SearchBoxControl.cpp

```diff
@@ -30,26 +30,34 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
     }
 
     // Method Description:
-    // - Getter for _goForward
+    // - Check if the current search direction is forward
     // Arguments:
     // - <none>
     // Return Value:
-    // - bool: the value of _goForward
-    bool SearchBoxControl::GoForward()
+    // - bool: the current search direction, determined by the
+    //         states of the two direction buttons
+    bool SearchBoxControl::_GoForward()
     {
-        return _goForward;
+        if (GoForwardButton().IsChecked().GetBoolean())
+        {
+            return true;
+        }
+        else
+        {
+            return false;
+        }

```

Comment:
can you not just `return GoForwardButton().IsChecked().GetBoolean()` just like `_CaseSensitive` below?

Reply:
changed